docs/report/017.ipynb

```diff
@@ -967,20 +967,31 @@
    "outputs": [],
    "source": [
     "prod_couplings = {}\n",
-    "for res in resonance_choices[1]:\n",
-    "    i = stringify(res)\n",
-    "    prod_couplings[H_prod[i, 0, -half]] = 1\n",
-    "    prod_couplings[H_prod[i, -1, -half]] = 1 - 1j\n",
-    "    prod_couplings[H_prod[i, +1, +half]] = -3 - 3j\n",
-    "    prod_couplings[H_prod[i, 0, +half]] = -1 - 4j\n",
-    "for res in resonance_choices[2]:\n",
-    "    i = stringify(res)\n",
-    "    prod_couplings[H_prod[i, +half, 0]] = 1\n",
-    "    prod_couplings[H_prod[i, -half, 0]] = 2j\n",
-    "for res in resonance_choices[3]:\n",
-    "    i = stringify(res)\n",
-    "    prod_couplings[H_prod[i, +half, 0]] = -7 + 3j\n",
-    "    prod_couplings[H_prod[i, -half, 0]] = -13 + 5j\n",
+    "#\n",
+    "# chain 23:\n",
+    "prod_couplings[H_prod[Str(\"K^{*}(892)^{0}\"), 0, -half]] = 1\n",
+    "prod_couplings[H_prod[Str(\"K^{*}(892)^{0}\"), -1, -half]] = 1 - 1j\n",
+    "prod_couplings[H_prod[Str(\"K^{*}(892)^{0}\"), +1, +half]] = -3 - 3j\n",
+    "prod_couplings[H_prod[Str(\"K^{*}(892)^{0}\"), 0, +half]] = -1 - 4j\n",
+    "#\n",
+    "prod_couplings[H_prod[Str(\"K_{0}^{*}(1430)^{0}\"), 0, -half]] = 1\n",
+    "prod_couplings[H_prod[Str(\"K_{0}^{*}(1430)^{0}\"), -1, -half]] = 1 - 1j\n",
+    "prod_couplings[H_prod[Str(\"K_{0}^{*}(1430)^{0}\"), +1, +half]] = -3 - 3j\n",
+    "prod_couplings[H_prod[Str(\"K_{0}^{*}(1430)^{0}\"), 0, +half]] = -1 - 4j\n",
+    "#\n",
+    "prod_couplings[H_prod[Str(\"K_{2}^{*}(1430)^{0}\"), 0, -half]] = 1\n",
+    "prod_couplings[H_prod[Str(\"K_{2}^{*}(1430)^{0}\"), -1, -half]] = 1 - 1j\n",
+    "prod_couplings[H_prod[Str(\"K_{2}^{*}(1430)^{0}\"), +1, +half]] = -3 - 3j\n",
+    "prod_couplings[H_prod[Str(\"K_{2}^{*}(1430)^{0}\"), 0, +half]] = -1 - 4j\n",
```

Comment:
Eventually you want to remove the K*(1430) resonances, no? https://github.com/ComPWA/compwa-org/pull/129#issuecomment-1105081621

Reply:
yes. Eventually. Just did not do all at once